source/utilities/time/test.js

```diff
@@ -170,6 +171,17 @@ describe("Testing time utilities", () => {
     });
   });
 
+  describe("formatMonthYearShort()", () => {
+    const {formatMonthYearShort} = util;
+    it("Generates a DD de MMM YY format", () => {
```

Comment:
This text is the same from `formatDayMonthYearShort`

Reply:
True. Now is fixed.
Thanks